plugin/trino-ranger/src/main/java/io/trino/plugin/ranger/RangerSystemAccessControlFactory.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.ranger;
+
+import com.google.inject.Injector;
+import com.google.inject.Scopes;
+import io.airlift.bootstrap.Bootstrap;
+import io.trino.spi.security.SystemAccessControl;
+import io.trino.spi.security.SystemAccessControlFactory;
+
+import java.util.Map;
+
+import static com.google.common.base.Throwables.throwIfUnchecked;
+import static io.airlift.configuration.ConfigBinder.configBinder;
+import static java.util.Objects.requireNonNull;
+
+public class RangerSystemAccessControlFactory
+        implements SystemAccessControlFactory
+{
+    private static final String NAME = "ranger";
```

Comment:
There are many ranger plugins. I could imagine that in future one could implement a `SystemAccessControlFactory` for Hive policies in Ranger to be obeyed in Trino. To make this name more future-proof I would suggest to use `ranger-trino`. So in future one could introduce `ranger-hive`. WDYT?

Reply:
An implementation of `SystemAccessControl` is used by Trino to authorize operations i.e. not just operations on a specific catalog. Hence name like ranger-hive does not seen appropriate in this context.